src/features/User/Ui/Review/Review.tsx

```diff
@@ -1,9 +1,66 @@
-import React from 'react'
+import React from "react";
+import {
+  Grid,
+  Paper,
+  Typography,
+  Container,
+  ScopedCssBaseline,
+} from "@mui/material";
+import { styled } from "@mui/system";
+import Box from "@mui/material/Box";
+import styles from "./Review.module.scss";
+import img from "../../../../assets/images/review.png";
+import StartBooking from "../StartBooking/StartBooking";
+import Stack from "@mui/material/Stack";
+import StarIcon from "@mui/icons-material/Star";
+import { amber } from "@mui/material/colors";
 
+const AccentStarIcon = styled(StarIcon)({
+  color: amber[500],
+});
 const Review = () => {
   return (
-    <div>Review</div>
-  )
-}
+    <ScopedCssBaseline 
+    sx={{ 
+      // backgroundColor: "#f58c01" ,
+     paddingTop:"3rem",paddingBottom:"4rem"}}>
+      <Container 
+      maxWidth="md">
+        <Stack
+          spacing={{ xs: 1, sm: 2 }}
+          direction={{ xs: "column", md: "row" }}
+          useFlexGap
+          // flexWrap="wrap"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Box className={styles.reviewImageParent}>
```

Comment:
The Image banner should the same dimensions as the image
You can create a div that will wrap the image and make the image take its dimensions and then move the image using `transform:translate()`.

Here is an example of what I mean:
https://codepen.io/y-ahmed-ali-y/pen/gOEWLBb

Reply:
done